### ISSUE TITLE
Toolkit bot aggressive at closing items

### ISSUE THREAD

**Issue by michael-hawker** [nmetulev/github-bot-uwp-toolkit#19]
Related to #4.

I feel our toolkit bot closes items pretty aggressively, as usually we can work on things over the course of a month or two without update.  Or we have things in our backlog.

I feel like we should make the following improvements:

- [ ] If issue isn't assigned to a milestone, ignore?
- [ ] If issue isn't assigned to anyone, ignore?
- [ ] Increase inactivity timeout from 14 to 32 days?  Or maybe just keep pinging but not auto-close?

Thoughts?

**Comment by nmetulev** [comment]
I agree, the bot should not close issues. However, by default, it should be pinging all issues unless specifically tagged (such as feature requests), to avoid issues falling through the cracks. 32 days sounds too long though, I would keep it to 14.

**Comment by michael-hawker** [comment]
Yeah, I think part of the problem is it clutters up notifications, as suddenly all these issues get marked so it can make it hard to see an actual update to some other issue.

Maybe we can create some better heuristic based on assignee, milestone, and labels to help the bot know what course of action to take and how much to ping.  Like if we've labelled an issue but haven't assigned it to a milestone, we don't need to be pinging about it (assuming we audit our milestones effectively).
